### DISCUSSION TITLE
Move asdf-azure-cli to asdf-community

### DISCUSSION THREAD

**Discussion by jlarvie** [asdf-community/.github#55] [General]
The current asdf-azure-cli maintainer has archived the repo https://github.com/itspngu/asdf-azure-cli and is offering anyone to maintain it.  Seems like a good fit in asdf-community.

**Comment by smorimoto** [comment]
Consult this discussion post: https://github.com/orgs/asdf-community/discussions/57 This is the smoothest way to go. Of course I can do all the work for you, but it's done on a best effort basis. (I've been busy with work lately...)
